nursery/detect-mouse-movement-analysis.yml

```
@@ -0,0 +1,64 @@
+rule:
+  meta:
+    name: detect mouse movement analysis
+    namespace: anti-analysis/anti-vm/vm-detection
+    authors:
+      - tevajdr@gmail.com
+    description: Identifies code that analyzes mouse movement patterns to detect non-human/sandbox behavior. - Advanced pattern analysis using vector mathematics - Simple position tracking over time in loop - Correlation of mouse movement with click event
```

```suggestion
    description: Identifies code that analyzes mouse movement patterns to detect non-human/sandbox behavior
```

I suggest removing these to keep the description concise, and since you cover these below.